src/Data/Functor/Linear/Internal/Functor.hs

```diff
@@ -17,9 +26,12 @@ import qualified Control.Monad.Trans.Reader as NonLinear
 import qualified Control.Monad.Trans.State.Strict as Strict
 import Data.Functor.Compose
 import Data.Functor.Const
-import Data.Functor.Identity
+import Data.Functor.Identity (Identity (..))
```

Comment:
I'd rather we avoid explicit imports. If there are conflicts, I prefer using `hiding`.

Reply:
Fine, whatever.